components/header.js

```diff
@@ -84,6 +85,7 @@ function NotificationBell () {
 }
 
 function NavProfileMenu ({ me, dropNavKey }) {
+  const { registration: swRegistration, togglePushSubscription } = useServiceWorker()
```

Comment:
Is it safe to destructure this? Is it always defined?

We have some folks on iphone 7 (ttps://stacker.news/items/328115/r/k00b) which uses a really old safari so just wanted to double check

Reply:
Yes, should be always defined since `ServiceWorkerProvider` always returns an object with these keys:

```js
const contextValue = useMemo(() => ({
  registration,
  support,
  permission,
  requestNotificationPermission,
  togglePushSubscription
}), [registration, support, permission, requestNotificationPermission, togglePushSubscription])
```

Reply:
Sweet thanks!